@xen-orchestra/lite/src/views/vm/VmConsoleView.vue

```diff
@@ -1,7 +1,13 @@
 <template>
   <div v-if="!isReady">Loading...</div>
   <div v-else-if="!isVmRunning">Console is only available for running VMs.</div>
-  <RemoteConsole v-else-if="vmConsole" :location="vmConsole.location" />
+  <RemoteConsole
+    v-else-if="vmConsole"
+    :location="vmConsole.location"
+    :is-console-available="
+      !isOperationsPending(vm, STOP_OPERATIONS)
+    "
+  />
```

Comment:
ESLint is complaining here ^^

```suggestion
  <RemoteConsole
    v-else-if="vmConsole"
    :is-console-available="!isOperationsPending(vm, STOP_OPERATIONS)"
    :location="vmConsole.location"
  />
```